src/evolutions/src/main/scala/play/api/db/slick/evolutions/EvolutionsModule.scala

```diff
@@ -21,7 +21,7 @@ class EvolutionsModule extends Module {
  */
 trait SlickEvolutionsComponents {
   @deprecated("Use slickApi instead", "3.0.0")
-  def api: SlickApi = slickApi
+  def api: SlickApi
```

Comment:
@marcospereira I don't think this is the right way. How would you do it? I don't know.